### ISSUE TITLE
Ability to add videos from other sites

### ISSUE THREAD

**Issue by mgogoulos** [plumi/plumi.app#69]
**Reported by and on 5 Jul 2007 16:16 UTC**
Plone4artists video allows you to take videos from Google Video, YouTube, blip.tv, etc and add them as an item.

http://plone4artists.org/products/plone4artistsvideo

It would be good to create a specific content type/product for adding this kind of videos to enable broader sharing of content.

**Comment by mgogoulos** [comment]
**Comment by anonymous on 19 Oct 2007 06:35 UTC**
Talking to someone from foe today they were looking at updating to the latest version of kupu which allows embedded videos. This is a bit simpler than what I was envisioning but could be good to have available.

Other possibilities include 
http://plone.org/products/atgooglevideo/

**Comment by mgogoulos** [comment]
**Modified by and on 29 Nov 2009 08:46 UTC**

**Comment by mgogoulos** [comment]
**Modified by and on 21 Apr 2010 03:50 UTC**

**Comment by mgogoulos** [comment]
**Modified by and on 21 Apr 2010 05:48 UTC**

**Comment by mgogoulos** [comment]
**Modified by anna on 26 Oct 2010 13:15 UTC**

**Comment by mgogoulos** [comment]
**Modified by and on 26 Oct 2010 13:56 UTC**

**Comment by mgogoulos** [comment]
**Modified by and on 28 Oct 2010 10:54 UTC**

**Comment by mgogoulos** [comment]
**Comment by mike on 10 Nov 2010 10:07 UTC**
we need to think about how videos hosted on external sites will be displayed for the user.

the viewer of these videos will have a different experience to viewing a normal video - you will not be able to download it. we need to then explain in the UI that this is the case...

or should videos hosted externally be quite separate from internal videos?

- anna

**Comment by mgogoulos** [comment]
**Comment by anna on 2 Dec 2010 11:37 UTC**
Ok, we discussed how these videos might work. The proposal has been added to the ticket description.

**Comment by mgogoulos** [comment]
**Modified by anna on 2 Dec 2010 11:43 UTC**

**Comment by mgogoulos** [comment]
**Comment by anna on 3 Dec 2010 01:21 UTC**
Mike suggests using the same PlumiVideo content type, with option to add link / embed code instead of upload a file (but still be able to differentiate these videos from normal videos, and thus list them differently on the site), and also to split off the changes to publish form into another ticket.

Ticket for changes to publish form:
http://plumi.org/ticket/529

**Comment by mgogoulos** [comment]
**Comment by anna on 15 Dec 2010 11:46 UTC**
Moving this to the re-factoring milestone, as Unweb advise that this should be done after changing Plumi video content type into schema extender

**Comment by mgogoulos** [comment]
**Modified by anna on 28 Jan 2011 04:40 UTC**

**Comment by mgogoulos** [comment]
**Comment by dimo on 13 Apr 2011 16:26 UTC**
This is not something that can be completed in this milestone & contract. My suggestion is to do it in parallel with the video publish form redesign

**Comment by mgogoulos** [comment]
**Comment by and on 15 Apr 2011 15:09 UTC**
I will convert the remaining 4.2 tickets to a 4.3 milestone and shift the rest back a version also. Major means it won't actually go in 4.2.

**Comment by mgogoulos** [comment]
**Modified by dimo on 28 Sep 2011 17:54 UTC**
